### PR TITLE
[RW-11668][risk=no] Link to SAS support article from SAS cloud env page

### PR DESCRIPTION
**Pull request by NehaBroad**
Adding links for SAS and 
Extracting support note section :
Screenshot 2024-03-12 at 1 19 47 PM
Screenshot 2024-03-12 at 1 20 02 PM
Screenshot 2024-03-12 at 1 20 10 PM

---
**PR checklist**

- [x] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [x] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [x] I have manually tested this change and my testing process is described above.
- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [ ] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [ ] None of the above apply to this change.

### CODE REVIEW

File: ui/src/app/components/common-env-conf-panels/support-note-section.tsx
Comment (evrii) [inline ui/src/app/components/common-env-conf-panels/support-note-section.tsx:1]: Great idea! I love that you pulled this out into its own component.

File: ui/src/app/utils/aou_external_links.tsx [hunk 1]
```diff
@@ -2,3 +2,8 @@ export const CROMWELL_INTRO_LINK =
   'https://cromwell.readthedocs.io/en/stable/tutorials/FiveMinuteIntro/';
 export const WORKFLOW_AND_WDL_LINK =
   'https://support.terra.bio/hc/en-us/articles/360037117492-Overview-Getting-started-with-WDL';
+
+export const SAS_INTRO_LINK =
```
Comment (evrii) [inline ui/src/app/utils/aou_external_links.tsx:6]: I think that these urls would be better placed in:
https://github.com/all-of-us/workbench/blob/046ad35626405c4df1f7683d3fc8cb61b76f6c60/ui/src/app/utils/zendesk.ts

**Review by evrii — APPROVED** [review]
Looks good. I verified that RStudio, Cromwell, and SAS all display as expected.

File: ui/src/app/utils/aou_external_links.tsx (hunk 1 above)
Comment (NehaBroad) [reply ui/src/app/utils/aou_external_links.tsx:6]: Yeah thats what i was thinking but the urls doesnt seem to have zendesk pattern, will follow up  offline

**Review by evrii — APPROVED** [review]
Looks good!

**Pull request merged**